cli/src/pixl_cli/_io.py

```diff
@@ -160,6 +161,7 @@ class DF_COLUMNS(StrEnum):  # noqa: N801
     "PrimaryMrn": "mrn",
     "AccessionNumber": "accession_number",
     "procedure_date": "study_date",
+    "StudyUid_X": "study_uid",
```

Comment:
might be missing it but we should edit our test CSV files and parquet files so that they have a `study_uid` column

Reply:
also seeing it on line `142`, we should also make sure that we filter out any rows where the accession number is an empty string 🙏🏻

Reply:
> might be missing it but we should edit our test CSV files and parquet files so that they have a `study_uid` column

Ah yes, I assumed we would get some new test data from an OMOP-ES run. How did we generate the original test data?